tasks/ataru.rake

```diff
@@ -0,0 +1,6 @@
+namespace :ataru do
+  desc 'Check if the code snippets in our markdown files are working as expected'
+  task :check do
+    system('bundle exec ataru check') || exit(1)
```

Comment:
maybe you want [`Bundle.with_clean_env` like how Rails does it?](https://github.com/rails/rails/blob/b2e34d2fc26ea176758ba1f9901f8fa518b8655e/railties/lib/rails/generators/app_base.rb#L302)?

```ruby
      def bundle_command(command)
        say_status :run, "bundle #{command}"

        # We are going to shell out rather than invoking Bundler::CLI.new(command)
        # because `rails new` loads the Thor gem and on the other hand bundler uses
        # its own vendored Thor, which could be a different version. Running both
        # things in the same process is a recipe for a night with paracetamol.
        #
        # We use backticks and #print here instead of vanilla #system because it
        # is easier to silence stdout in the existing test suite this way. The
        # end-user gets the bundler commands called anyway, so no big deal.
        #
        # We unset temporary bundler variables to load proper bundler and Gemfile.
        #
        # Thanks to James Tucker for the Gem tricks involved in this call.
        _bundle_command = Gem.bin_path('bundler', 'bundle')

        require 'bundler'
        Bundler.with_clean_env do
          output = `"#{Gem.ruby}" "#{_bundle_command}" #{command}`
          print output unless options[:quiet]
        end
      end
bundle_command "exec ataru check"
```
(and maybe see https://github.com/troessner/reek/pull/540#issuecomment-112193618 ? )